build.ps1

```diff
@@ -26,13 +26,16 @@ function build-partial() {
         //:git-revision `
         //compiler/daml-lf-ast/... `
         //daml-lf/interface/... `
-        //language-support/java/bindings/...
+        //language-support/java/bindings/... `
+        //navigator/backend/... `
```

Comment:
Is there anything in `//navigator` that doesn’t build yet? If not we could just use `//navigator/...`.

Reply:
Yes, integration-tests reference `damlc-dist`

Reply:
If it helps, the integration test needs something to compile DAML to DAR, but we can change how exactly this is achieved.
I see:
- https://github.com/digital-asset/daml/blob/master/navigator/integration-test/src/main/scala/com/digitalasset/navigator/test/runner/PackagedDamlc.scala, using `com.digitalasset.damlc.Runner`
- https://github.com/digital-asset/daml/blob/master/navigator/integration-test/src/main/scala/com/digitalasset/navigator/test/runner/HeadDamli.scala, using `bazel run damlc`
- I think I saw I some other project that compiled DAML to DAR at compile time, and included the DAR file as a resource.

The different files at `navigator/test/runner` are from before we had head based development and should be cleaned up.